app/build.gradle

```diff
@@ -84,6 +84,10 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    lintOptions {
+        lintConfig file('lint.xml')
+    }
```

Comment:
we should have the `InvalidPackage` rules added to the common lintConfig file and incorporate it into the static analysis plugin instead of override it here